summingbird-storm/src/main/scala/com/twitter/summingbird/storm/collector/KeyValueOutputCollector.scala

```diff
@@ -0,0 +1,30 @@
+package com.twitter.summingbird.storm.collector
+
+import backtype.storm.spout.SpoutOutputCollector
+import java.util.{ List => JList }
+
+/**
+ * Created by pnaramsetti on 7/19/16.
+ *
+ * The KeyValueOutputCollector is used to unwrap the Value object when passed on by the Spout.
+ */
+class KeyValueOutputCollector(self: SpoutOutputCollector, func: JList[AnyRef] => JList[AnyRef]) extends SpoutOutputCollector(null) {
+
+  override def emitDirect(i: Int, s: String, list: JList[AnyRef], o: scala.Any): Unit = self.emitDirect(i, s, func(list), o)
```

Comment:
I think you want `AnyRef` here, not `Any`. Java `Object` is more correctly `AnyRef` though the compiler might not complain here, we should not coerce from `Any`.